packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -419,8 +419,10 @@ class ImportedDatabaseCluster extends DatabaseClusterBase implements IDatabaseCl
 export interface DatabaseClusterProps extends DatabaseClusterBaseProps {
   /**
    * Username and password for the administrative user
+   *
+   * @default - A username of 'admin' and SecretsManager-generated password
    */
-  readonly masterUser: Login;
+  readonly masterUser?: Login;
```

Comment:
Gotta say, I'm not a huge fan of the `masterUser` name... How about we take this opportunity to rename it to something like `login`? (If you have a better name, I'm all ears 🙂)

Or, do our usual "deprecate + new field" dance if you want to minimize the breaking changes :)

Reply:
I do like `login` better, and was briefly tempted to do a rename, but held off as I liked that without that, there was a degree of backwards compatibility here. If users with clusters are able to keep their existing code -- and depending on language and specifics of how it's written, lots of users will be able to -- than that's a huge perk.

The "deprecate + new field" dance is an acceptable compromise though; I can do that throughout.